stepruncmd/timeoutcmd/timeoutcmd.go

```diff
@@ -62,15 +60,6 @@ func (c *Command) SetStandardIO(in io.Reader, out, err io.Writer) {
 
 // Start starts the command run.
 func (c *Command) Start() error {
-	// setting up notification for signals, so we can have separated logic to end the process
```

Comment:
For future readers/reviewers of this PR:

This method is called when a step is requested to run, having the interrupt handling logic here prevented us from a proper cleanup due it issuing an os.Exit before we could handle it outside as well.

Even though this is removed, we have implemented the signal handle in [run.go](https://github.com/bitrise-io/bitrise/pull/908/files#diff-d8b7e73d77fa79e1493c5173dbf6914d4048b87c83ae9edbb97ca3bd5593bc47R77). That will properly clean up containers before killing the process, so the end result should be the same.